### PR TITLE
config: add workflow config and deployment config

### PR DESCRIPTION
**Pull request by Mini256**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,6 +11,8 @@ COPY package.json package-lock.json ./
 RUN npm ci
```
Comment (hi-rustin) [inline Dockerfile:11]: Why we need copy it twice?

**Review by hi-rustin — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (Mini256) [reply Dockerfile:11]: The first is to install dependencies to build, and the second is to install runtime environment, `npm start` use the `probot` to run.

Comment (Mini256) [reply Dockerfile:11]: Maybe we can build the project in host and just run in container?
image

Comment (hi-rustin) [reply Dockerfile:11]: But I think we only need copy these files once.

Comment (hi-rustin) [reply Dockerfile:11]: You can refer to https://github.com/ti-community-infra/ti-community-bot/blob/master/Dockerfile

Comment (Mini256) [reply Dockerfile:11]: But if we can remove the dependency of development environment from the image, can we reduce the space occupied by image

Comment (Mini256) [reply Dockerfile:11]: If we compile on the GitHub action machine, we can also make good use of the GitHub cache.

Comment (Mini256) [reply Dockerfile:11]:
```
FROM node:12-slim

WORKDIR /usr/src/app

COPY package.json package-lock.json ./

RUN npm ci --production
RUN npm cache clean --force

COPY ./lib lib
COPY ormconfig.js ./

ENV NODE_ENV="production"
EXPOSE 3000

CMD [ "npm", "start" ]
```

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**